Cargo.toml

```diff
@@ -1,32 +1,7 @@
-[package]
-name = "centrifuge-chain"
```

Comment:
I would also keep this and name the node crate `centrifuge-node`

Reply:
The workspace can not have a `[package]`. It is related to a crate and now the root `Cargo.toml` does not identify any crate